src/components/WalletDropdown/DefaultMenu.tsx

```diff
@@ -0,0 +1,263 @@
+import { Trans } from '@lingui/macro'
+import { CurrencyAmount, Token } from '@uniswap/sdk-core'
+import { useWeb3React } from '@web3-react/core'
+import { getConnection } from 'connection/utils'
+import { getChainInfoOrDefault } from 'constants/chainInfo'
+import { SupportedChainId } from 'constants/chains'
+import { useActiveLocale } from 'hooks/useActiveLocale'
+import useCopyClipboard from 'hooks/useCopyClipboard'
+import useStablecoinPrice from 'hooks/useStablecoinPrice'
+import useNativeCurrency from 'lib/hooks/useNativeCurrency'
+import { useCallback, useMemo } from 'react'
+import { ChevronRight, Copy, ExternalLink, Moon, Power, Sun } from 'react-feather'
+import { Text } from 'rebass'
+import { useToggleWalletModal } from 'state/application/hooks'
+import { useCurrencyBalanceString } from 'state/connection/hooks'
+import { useAppDispatch } from 'state/hooks'
+import { useDarkModeManager } from 'state/user/hooks'
+import { updateSelectedWallet } from 'state/user/reducer'
+import styled from 'styled-components/macro'
+
+import useENS from '../../hooks/useENS'
+import { shortenAddress } from '../../nft/utils/address'
+import { useToggleModal } from '../../state/application/hooks'
+import { ApplicationModal } from '../../state/application/reducer'
+import { useUserUnclaimedAmount } from '../../state/claim/hooks'
+import { useAllTransactions } from '../../state/transactions/hooks'
+import { ButtonPrimary } from '../Button'
+import StatusIcon from '../Identicon/StatusIcon'
+import IconButton, { IconHoverText } from './IconButton'
+import { MenuState } from './index'
+
+const UNIbutton = styled(ButtonPrimary)`
+  background: linear-gradient(to right, #9139b0 0%, #4261d6 100%);
+  border-radius: 12px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  margin-top: 12px;
+  color: white;
+`
+
+const Column = styled.div`
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+`
+
+const ConnectButton = styled.button`
+  border: none;
+  outline: none;
+  border-radius: 12px;
+  height: 44px;
+  width: 288px;
+  background-color: ${({ theme }) => theme.accentAction};
+  color: white;
+  font-weight: 600;
+  font-size: 16px;
+  cursor: pointer;
+`
+
+const IconContainer = styled.div`
+  display: inline-block;
+  float: right;
+  & > a {
+    margin-right: 8px;
+  }
+  & > a:last-child {
+    margin-right: 0px;
+    ${IconHoverText}:last-child {
+      left: 0px;
+    }
+  }
+`
+
+const Divider = styled.div`
+  border: 0.5px solid ${({ theme }) => theme.backgroundOutline};
```

Comment:
somewhat concerned with using a .5px border, could you just confirm with fred that this is expected?